dask/dataframe/utils.py

```diff
@@ -11,7 +11,7 @@
 
 import numpy as np
 import pandas as pd
-from pandas.api.types import is_categorical_dtype, is_dtype_equal
+from pandas.core.dtypes.api import is_categorical_dtype, is_dtype_equal
```

Comment:
It was right before:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.api.types.is_categorical_dtype.html?highlight=is_categorical_dtype

Public documentation wins over type annotations, generally.
If mypy complains, then it's a bug in the pandas typing annotations and it should be reported.

Reply:
Mypy isn't complaining any more. Reverted this.